test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,48 @@
+LD_FLAGS=$1
+IMAGE_VERSION=$2
+EKS_CLUSTER_NAME=$3
+
+AWS_REGION="us-west-2"
+AWS_ACCOUNT_ID=$(aws --region $AWS_REGION sts get-caller-identity --query Account --output text)
+
+ECR_REGISTRY_NAME="csisecretdriver"
+ECR_REGISTRY_URL="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com"
+ECR_REGISTRY_URL_WITH_NAME="$ECR_REGISTRY_URL/$ECR_REGISTRY_NAME"
```

Comment:
These variables are defined in the `Makefile` and here. We need this only in the script?

Reply:
Yup some of these are only needed in the script. Will update.